dispatch/static/manager/src/js/components/ContentEditor/embeds/VideoEmbed.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+
+import { FormInput, TextInput } from '../../inputs'
+
+class VideoEmbedComponent extends React.Component {
+
+  constructor(props){
+    super(props)
+
+    this.handleUrlChange = this.handleUrlChange.bind(this)
```

Comment:
pretty sure you don't have to add the bind if you [wrote the functions using () => syntax ](https://medium.com/@machnicki/handle-events-in-react-with-arrow-functions-ede88184bbb#.2dlb31qok)

```
handleUrlChange = (e) => {
   ....
}
```